### ISSUE TITLE
simultaneous keyboard interaction doesn't work with screen reader

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#61]
Discovered during design meeting today, w/s and arrow keys don't work on `RatioInteractionListener` when NVDA is turned on. Here is what the Node currently looks like:

```js

    const a11yRatioContainer = new Node( {
      tagName: 'div',
      role: 'application',
      focusable: true,
      labelContent: 'Ratio:',
      children: [
        this.leftRatioHalf,
        this.rightRatioHalf
      ]
    } );
```

I would have expected that the role and focusable options were the only thing necessary to make this work.

**Comment by zepumph** [comment]
Fixed by using the right option, oops. Thanks for the help @jessegreenberg.
